app/models/data_element.rb

```diff
@@ -6,7 +6,11 @@ class DataElement < ApplicationRecord
 
   belongs_to :concept, inverse_of: :data_elements
 
-  translates :description
+  def description
+    return description_cy if I18n.locale == :cy
+
+    description_en
+  end
```

Comment:
Should this be a view helper rather than on the model? We don't have an example, so I'm interested to think where any other things like this might end up living.

Reply:
This is to mimick the `globalize` gem behaviour, so that we won't have to handle `DataElement` objects any different from `Category` and `Concept` objects.
Happy to move it into a view helper if you think it's a better choice, or to have a chat about what the best approach is.